cmd/containerd-shim-runhcs-v1/options/runhcs.proto

```diff
@@ -64,10 +64,25 @@ message Options {
 	// when a container requests a gpu
 	string GPUVHDPath = 10;
 
-        // scale_cpu_limits_to_sandbox indicates that container CPU limits should
-        // be adjusted to account for the difference in number of cores between the
-        // host and UVM.
-        bool scale_cpu_limits_to_sandbox = 11;
+	// scale_cpu_limits_to_sandbox indicates that container CPU limits should
+	// be adjusted to account for the difference in number of cores between the
+	// host and UVM.
+	bool scale_cpu_limits_to_sandbox = 11;
+
+	// default_container_scratch_size_in_gb is the default scratch size (sandbox.vhd) 
+	// to be used for containers. Every container will get a sandbox of `size_in_gb` assigned
+	// instead of the default of 20GB. 
+	int32 default_container_scratch_size_in_gb = 12;
+
+	// default_vm_scratch_size_in_gb is the default scratch size (sandbox.vhd) 
+	// to be used for the UVM. This only applies to WCOW as LCOW doesn't mount a scratch 
+	// specifically for the UVM.
+	int32 default_vm_scratch_size_in_gb = 13;
+
+	// share_scratch specifies if we'd like to reuse scratch space between multiple containers. 
+	// This currently only affects LCOW. The sandbox containers scratch space is re-used for all
+	// subsequent containers launched in the pod.  
+	bool share_scratch = 14;
```

Comment:
Why are we adding in these options here but then never using them in this PR? I'd like to see this moved to another PR if possible.

Reply:
They're config options for containerd that will be used for the snapshotter. Theres nothing we can do with the values once we receive them in the shim, they need to be applied/used at the cri layer, they're useless here. Theres no other PR these could go in.